sql/src/main/java/io/crate/planner/operators/Count.java

```diff
@@ -89,10 +89,11 @@ public ExecutionPlan build(PlannerContext plannerContext,
             plannerContext.nextExecutionPhaseId(),
             COUNT_PHASE_NAME,
             countPhase.nodeIds().size(),
+            1,
             Collections.singletonList(plannerContext.handlerNode()),
             Collections.singletonList(DataTypes.LONG),
             Collections.singletonList(MergeCountProjection.INSTANCE),
-            DistributionInfo.DEFAULT_SAME_NODE,
+            DistributionInfo.DEFAULT_BROADCAST,
```

Comment:
also here, why needed? afaik the default should stay same node, only if used intermediate it should change.

Reply:
Good point. No need to change the default here. Instead, we should adapt the distribution info in case it cannot be optimized same-node.